### PR TITLE
Update Project-Glossary.md

### PR DESCRIPTION
**Pull request by lfrohlich**
Updated some glossary items and deleted a few. Could use more revisiting at some point.

### CODE REVIEW

File: docs/Background/Project-Glossary.md [hunk 1]
```diff
@@ -32,13 +32,10 @@
 
 **Office of Planning, Research, and Evaluation (OPRE)** - An ACF office that focuses on increasing the quality, usefulness, sharing, and analysis of data to improve ACF programs and program participants’ outcomes. Sibling to OFA.
 
-**PIR** -
-
 **Sample data** -
```
Comment (ADPennington) [inline docs/Background/Project-Glossary.md:36]:
```suggestion
STTs (see definition below) may elect to report TANF information for all families served by the jurisdiction's TANF program or for a stratified sample of families served in the jurisdiction. Sample data, herein, is a reference to data from STTs that submit information for a stratified sample. As of FY2021, 22 STTs submit sample data.
```

**Review by ADPennington — COMMENTED** [review]
👍🏾 added a suggestion re: sample data description.

**Comment by lfrohlich** [comment]
> 👍🏾 added a suggestion re: sample data description.

Good add!

**Review by kniz-raft — COMMENTED** [review]
@lfrohlich I noticed that "Universe Data" doesn't have a definition, do you think that is something we could add too?

File: docs/Background/Project-Glossary.md [hunk 2]
```diff
@@ -50,6 +47,4 @@
```
Comment (ADPennington) [inline docs/Background/Project-Glossary.md:47]:
```suggestion
STTs (see definition above) may elect to report TANF information for all families served by the jurisdiction's TANF program or for a stratified sample of families served in the jurisdiction. Universe data, herein, is a reference to data from STTs that submit information for **all families** served in the jurisdiction. 
```

**Comment by lfrohlich** [comment]
> @lfrohlich I noticed that "Universe Data" doesn't have a definition, do you think that is something we could add too?

Good call. Thanks @ADPennington  for the add!

**Review by ADPennington — APPROVED** [review]
👍🏾

**Pull request merged**